src/main/scala/scalaz/netty/Client.scala

```diff
@@ -75,7 +76,7 @@ private[netty] final class Client(limit: Int) {
 
     override def channelRead(ctx: ChannelHandlerContext, msg: AnyRef): Unit = {
       val buf = msg.asInstanceOf[ByteBuf]
-      val bv = ByteVector(buf.nioBuffer)       // copy data (alternatives are insanely clunky)
+      val bv = ByteVector.view(buf.nioBuffer)       // copy data (alternatives are insanely clunky)
       buf.release()
```

Comment:
Ah, I did not read the code closely - yes that view operation would be busted ...